src/components/Home/AmountInput.js

```diff
@@ -12,20 +12,22 @@ import colors from '../../styles/colors';
 
 export default function AmountInput({ value, error, maxLength, onChange, tintColor, autofocus }) {
 	return (
-		<View style={{ flexDirection: 'row' }}>
+		<View style={{ flexDirection: 'row', justifyContent: 'center' }}>
 			<TextInput
 				style={{
 					fontSize: 70,
 					fontWeight: 'bold',
 					color: error == null ? tintColor : colors.error,
 					alignSelf: 'center',
 					textAlign: 'right',
+					width: value ? 'auto' : 180,
+					padding: 0,
+					margin: 0,
 				}}
 				keyboardType="decimal-pad"
-				placeholder="00,00"
+				placeholder={value ? null : '00,00'}
```

Comment:
The default placeholder looks like this on iOS :
<img width="375" alt="image" src="https://user-images.githubusercontent.com/17085831/63060683-3199bc80-bef3-11e9-94ee-7537612953bb.png">

Reply:
Ok I go for `''` then